src/OperationalInsights/OperationalInsights/ChangeLog.md

```diff
@@ -19,13 +19,17 @@
 -->
 ## Upcoming Release
 
+## Version 1.2.1
```

Comment:
Please juet add the comment under 'Upcoming Releases'  Our tool will determine the next version

Reply:
sure